### ISSUE TITLE
Library cxf-rt-transports-http is not needed in 3.0.0-beta-5 anymore

### ISSUE THREAD

**Issue by PavelCibulka** [todvora/eet-client#24]
Remove **cxf-rt-transports-http** from 3.0.0-beta-5

**Comment by todvora** [comment]
Huh, it seems that we really don't need this lib. Thanks for the info!

**Comment by todvora** [comment]
Hm, not entirely true, the build is failing on JDK 6: https://travis-ci.org/todvora/eet-client/builds/204654006

Here is the dependencies tree:

```
tomas@desktop:~/projects/eet-client$ mvn dependency:tree 
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building eet-client 3.0.0-beta-5
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.1:tree (default-cli) @ eet-client ---
[INFO] cz.tomasdvorak:eet-client:jar:3.0.0-beta-5
[INFO] +- org.apache.cxf:cxf-rt-frontend-jaxws:jar:3.0.10:compile
[INFO] |  +- xml-resolver:xml-resolver:jar:1.2:compile
[INFO] |  +- org.apache.cxf:cxf-core:jar:3.0.10:compile
[INFO] |  |  +- org.codehaus.woodstox:woodstox-core-asl:jar:4.4.1:compile
[INFO] |  |  |  \- org.codehaus.woodstox:stax2-api:jar:3.1.4:compile
[INFO] |  |  \- org.apache.ws.xmlschema:xmlschema-core:jar:2.2.1:compile
[INFO] |  +- org.apache.cxf:cxf-rt-bindings-soap:jar:3.0.10:compile
[INFO] |  |  +- org.apache.cxf:cxf-rt-wsdl:jar:3.0.10:compile
[INFO] |  |  |  \- wsdl4j:wsdl4j:jar:1.6.3:compile
[INFO] |  |  \- org.apache.cxf:cxf-rt-databinding-jaxb:jar:3.0.10:compile
[INFO] |  |     +- com.sun.xml.bind:jaxb-impl:jar:2.2.11:compile
[INFO] |  |     \- com.sun.xml.bind:jaxb-core:jar:2.2.11:compile
[INFO] |  +- org.apache.cxf:cxf-rt-bindings-xml:jar:3.0.10:compile
[INFO] |  +- org.apache.cxf:cxf-rt-frontend-simple:jar:3.0.10:compile
[INFO] |  +- org.apache.cxf:cxf-rt-transports-http:jar:3.0.10:compile
[INFO] |  \- org.apache.cxf:cxf-rt-ws-addr:jar:3.0.10:compile
[INFO] |     \- org.apache.cxf:cxf-rt-ws-policy:jar:3.0.10:compile
[INFO] +- org.apache.cxf:cxf-rt-ws-security:jar:3.0.10:compile
[INFO] |  +- org.apache.cxf:cxf-rt-security:jar:3.0.10:compile
[INFO] |  |  \- org.apache.wss4j:wss4j-ws-security-common:jar:2.0.9:compile
[INFO] |  |     +- org.apache.santuario:xmlsec:jar:2.0.7:compile
[INFO] |  |     |  \- commons-codec:commons-codec:jar:1.10:compile
[INFO] |  |     +- org.opensaml:opensaml:jar:2.6.1:compile
[INFO] |  |     |  +- org.opensaml:openws:jar:1.5.1:compile
[INFO] |  |     |  |  \- org.opensaml:xmltooling:jar:1.4.1:compile
[INFO] |  |     |  \- joda-time:joda-time:jar:2.2:compile
[INFO] |  |     \- org.jasypt:jasypt:jar:1.9.2:compile
[INFO] |  +- net.sf.ehcache:ehcache:jar:2.8.5:compile
[INFO] |  +- org.apache.wss4j:wss4j-ws-security-dom:jar:2.0.9:compile
[INFO] |  +- org.apache.wss4j:wss4j-policy:jar:2.0.9:compile
[INFO] |  |  \- org.apache.neethi:neethi:jar:3.0.3:compile
[INFO] |  +- org.apache.wss4j:wss4j-ws-security-stax:jar:2.0.9:compile
[INFO] |  |  \- org.apache.wss4j:wss4j-bindings:jar:2.0.9:compile
[INFO] |  \- org.apache.wss4j:wss4j-ws-security-policy-stax:jar:2.0.9:compile
[INFO] +- org.bouncycastle:bcprov-jdk15on:jar:1.55:compile
[INFO] +- org.apache.logging.log4j:log4j-api:jar:2.3:compile
[INFO] +- org.apache.logging.log4j:log4j-core:jar:2.3:compile
[INFO] +- org.apache.logging.log4j:log4j-slf4j-impl:jar:2.3:compile
[INFO] |  \- org.slf4j:slf4j-api:jar:1.7.12:compile
[INFO] \- junit:junit:jar:4.12:test
[INFO]    \- org.hamcrest:hamcrest-core:jar:1.3:test

```

**Comment by todvora** [comment]
Closing, the dependency is needed, removing cause only failures on jdk 1.6 and the lib will be included automatically in jdk 1.8 and 1.8.
